examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java

```diff
@@ -65,21 +83,43 @@ public KubernetesSeedProvider(Map<String, String> params) {
 			logger.warn("Seed provider couldn't lookup host " + host);
 		    }
 	    }
-    } 
+	// TODO: Load the CA cert when it is available on all platforms.
+	trustAll = new TrustManager[] {
+	    new X509TrustManager() {
+		public void checkServerTrusted(X509Certificate[] certs, String authType) {}
+		public void checkClientTrusted(X509Certificate[] certs, String authType) {}
+		public X509Certificate[] getAcceptedIssuers() { return null; }
+	    }
+	};
+	trustAllHosts = new HostnameVerifier() {
+		public boolean verify(String hostname, SSLSession session) {
+		    return true;
+		}
+	    };
+    }
 
     public List<InetAddress> getSeeds() {
         List<InetAddress> list = new ArrayList<InetAddress>();
-        String protocol = getEnvOrDefault("KUBERNETES_API_PROTOCOL", "http");
-        String hostName = getEnvOrDefault("KUBERNETES_RO_SERVICE_HOST", "localhost");
-        String hostPort = getEnvOrDefault("KUBERNETES_RO_SERVICE_PORT", "8080");
-
-        String host = protocol + "://" + hostName + ":" + hostPort;
+        String host = "https://kubernetes.default.cluster.local";
         String serviceName = getEnvOrDefault("CASSANDRA_SERVICE", "cassandra");
         String path = "/api/v1beta3/namespaces/default/endpoints/";
```

Comment:
I'm gonna merge this, but maybe in a followup you can fix this to get the namespace from the downward API? This forces the example to work only in the default namespace unnecessarily.